irohad/ametsuchi/impl/postgres_setting_query.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ametsuchi/impl/postgres_setting_query.hpp"
+
+#include <boost/lexical_cast.hpp>
+#include "interfaces/common_objects/types.hpp"
+#include "logger/logger.hpp"
+
+using namespace iroha::ametsuchi;
+
+namespace {
+  template <typename T>
+  bool getValueFromDb(soci::session &sql,
+                      const shared_model::interface::types::SettingKeyType &key,
+                      T &destination,
+                      const logger::LoggerPtr &log) {
+    boost::optional<shared_model::interface::types::SettingValueType> value;
+
+    try {
+      sql << "SELECT setting_value FROM setting WHERE setting_key = :key",
+          soci::into(value), soci::use(key, "key");
+    } catch (const std::exception &e) {
+      log->error("Failed to execute query: {}", e.what());
+      return false;
+    }
+
+    if (value) {
+      try {
+        destination = boost::lexical_cast<T>(value.get());
+        return true;
+      } catch (const boost::bad_lexical_cast &e) {
+        log->error("Failed to get value: {}", e.what());
+        return false;
+      }
+    }
+    return false;
+  }
+}  // namespace
+
+PostgresSettingQuery::PostgresSettingQuery(std::unique_ptr<soci::session> sql,
+                                           logger::LoggerPtr log)
+    : psql_(std::move(sql)), sql_(*psql_), log_(std::move(log)) {}
+
+std::shared_ptr<const shared_model::validation::Settings>
+PostgresSettingQuery::get() {
+  std::shared_ptr<shared_model::validation::Settings> kDefaultSettings =
```

Comment:
kSuchNaming is used for static constants, which is misleading here
```suggestion
  std::shared_ptr<shared_model::validation::Settings> default_settings =
```

Reply:
Fixed